.github/scripts/bounties.js

```diff
@@ -0,0 +1,57 @@
+const { Octokit } = require("@octokit/core");
+const octokit = new Octokit({ auth: process.env.GITHUB_TOKEN });
+
+async function run() {
+  const githubContext = JSON.parse(process.env.GITHUB_CONTEXT);
+  const issueID = githubContext.event.issue.number;
+  const issueTitle = githubContext.event.issue.title;
+  const repoName = githubContext.event.repository.name;
+  const repoOwner = githubContext.event.repository.owner.login;
+
+  try {
+    switch (githubContext.event.action) {
+      case 'opened':
+        const body = githubContext.event.issue.body +
+          "\n\n<br /><hr>\nCare about this issue? Want to get it " +
+          "resolved sooner? If you are a " +
+          "<a href='https://www.mautic.org/become-a-member-of-mautic'>member " +
+          "of Mautic</a>, you can add some funds to the " +
+          "<a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> " +
+          "so that the person who completes this task can claim those funds once it is " +
+          "merged by a member of the core team!";
+        
+        await octokit.request('PATCH /repos/{owner}/{repo}/issues/{issue_number}', {
+          owner: repoOwner,
+          repo: repoName,
+          issue_number: issueID,
+          body: body,
+        });
+        break;
+    
+      case 'labeled':
+        const labelName = githubContext.event.label.name;
+
+        if (labelName === 'bounty') {
+          const comment =
+            "This issue has a bounty associated with it. Check the total available " +
+            "<a href'https://opencollective.com/mautic/projects/bounties/transactions'> " +
```

Comment:
I left this in so people can still go check the transactions list - but ideally we need to find a way to filter them by the issue. Future project, perhaps.